queries.sql

```diff
@@ -1,3 +1,16 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+SELECT * FROM animals WHERE neutered = true;
+
+SELECT * FROM animals WHERE name != 'Gabumon';
+
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] Great job so far. However, as specified in the project requirements kindly add screenshots of each of your query results to the pull request description. See below for more details on project requirements:
<img width="780" alt="Screenshot 2023-08-22 at 01 40 37" src="https://github.com/rodrigopenayo98/vet_clinic_rod/assets/107192076/7d84a677-285e-4df8-8c91-005cc5fff3b5">

Reply:
@Strangeal Great, I already added screenshots of each query!